packages/api/src/services/speech.ts

```diff
@@ -8,7 +10,10 @@ export const shouldSynthesize = async (
   userId: string,
   page: Page
 ): Promise<boolean> => {
-  if (page.pageType === PageType.File || !page.content) {
+  if (
+    contentReaderForPageType(page.pageType) === ContentReader.Web ||
```

Comment:
I think this should be `!==`

Reply:
OMG that would have been hard to catch in testing too